recipe/meta.yaml

```
@@ -1,31 +1,33 @@
 {% set name = "Shapely" %}
-{% set version = "1.8.4" %}
+{% set version = "2.0.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version.split('.post')[0] }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: a195e51caafa218291f2cbaa3fef69fd3353c93ec4b65b2a4722c4cf40c3198c
+  url: https://github.com/{{ name }}/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 6250e79ca299222a239ce44df613d228f1216c5cc31351217bf377b03d7be4e0
 
 build:
   number: 0
   # geos currently isn't available on s390x
-  skip: True  # [py<36 or s390x]
-  skip: True  # [win and vc<14]
+  skip: True  # [py<37 or s390x]
+  script: 
+    - set GEOS_INCLUDE_PATH=%LIBRARY_INC%  # [win]
+    - set GEOS_LIBRARY_PATH=%LIBRARY_LIB%  # [win]
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv 
 
 requirements:
   build:
     - {{ compiler('c') }}
   host:
     - python
-    - cython >=0.29.24,<3
-    - geos
-    - numpy   1.19  # [py<310]
-    - numpy   1.21  # [py>=310]
+    - cython 0.29.33
+    - numpy
+    - geos 3.9.1
```

```suggestion
    - geos {{ geos }}
```

Reason: geos is pinned in aggregate conda_build_config as this:
```
geos:
  - 3.8.0  # [not (osx and arm64)]
  - 3.9.1  # [osx and arm64]
```

---

Thanks for that info!